tests/unit_tests/sparse_blas/source/sparse_spsv_buffer.cpp

```diff
@@ -47,6 +47,9 @@ int test_spsv(sycl::device *dev, sparse_matrix_format_t format, intType m, doubl
         matrix_properties.find(oneapi::mkl::sparse::matrix_property::symmetric) !=
         matrix_properties.cend();
 
+    // Use a fixed seed for operations very sensible to the input data
```

Comment:
```suggestion
    // Use a fixed seed for operations very sensitive to the input data
```

Reply:
Reworded in https://github.com/oneapi-src/oneMKL/pull/527/commits/3b29e3292e8c158e90f86388174c3ede5853a174